2021-finalproject/README2.md

```diff
@@ -0,0 +1,385 @@
+# shell的简单命令
```

Comment:
This file name is meaningless, try a meaningful name

Reply:
I had changed the meaningless name. Thank you!